apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -378,15 +378,15 @@ static void tc_libc_timer_clock_daysbeforemonth(void)
 	for (month_iter = 1; month_iter < 12; month_iter++) {
 		prev_month = clock_daysbeforemonth(month_iter, FALSE);
 		cur_month = clock_daysbeforemonth(month_iter+1, FALSE);
-		TC_ASSERT_EQ("clock_daysbeforemonth", cur_month-prev_month, notleapyear_days[month_iter]);
+		TC_ASSERT_EQ("clock_daysbeforemonth", cur_month-prev_month, notleapyear_days[month_iter+1]);
```

Comment:
You need to put a space before and after '+' operator, though I do not like TizenRT coding style.